### ISSUE TITLE
libappindicator-WARNING **: Using '/tmp' paths in SNAP environment will lead to unreadable resources

### ISSUE THREAD

**Issue by mishuvs** [electron/electron-api-demos#461]
On opening the tray demo, the following warning is shown:

```
(electron:7813): libappindicator-WARNING **: 23:47:56.853: Using '/tmp' paths in SNAP environment will lead to unreadable resources
```

What does it mean? How to fix it?

* What operating system are you using?
Ubuntu 18.04

* What version of Node.js is on your system?
v12.13.0

**Comment by gammacray** [comment]
Did you ever find a solution? I'm getting the same errors with a different Snap and I can't figure out what it is.

**Comment by mishuvs** [comment]
@gammacray No, it leads to the tray icon not showing in the tray. However, I think it goes away when the app is packaged, and the tray icon shows successfully when a packaged app is installed.
